File: src/main/java/com/github/codingvampyre/drachenfest/repositories/ParticipantRepository.java
package com.github.codingvampyre.drachenfest.repositories;

import com.github.codingvampyre.drachenfest.domain.Account;
import org.springframework.data.jpa.repository.JpaRepository;

public interface ParticipantRepository extends JpaRepository<Account, Long> {
    public Account findByemail(String email);
}

File: src/main/java/com/github/codingvampyre/drachenfest/repositories/ParticipantRepository.kt
package com.github.codingvampyre.drachenfest.repositories

import com.github.codingvampyre.drachenfest.domain.Account
import org.springframework.data.jpa.repository.JpaRepository

interface ParticipantRepository : JpaRepository<Account, Long> {
	fun findByemail(email: String): Account?
}